provider/resource_keycloak_openid_client_test.go

```diff
@@ -859,3 +889,27 @@ resource "keycloak_openid_client" "client" {
 }
 	`, realm, clientId, standardFlowEnabled, implicitFlowEnabled, directAccessGrantsEnabled, serviceAccountsEnabled)
 }
+
+func TestAccKeycloakOpenidClient_AccessToken_basic(t *testing.T) {
```

Comment:
nitpick: typically the actual test cases are at the top of the file.  Could you move this near the existing test functions?

Reply:
Done :)